src/headerCells/SortableHeaderCell.tsx

```diff
@@ -30,18 +42,15 @@ export default function SortableHeaderCell<R, SR>({
   priority,
   children
 }: Props<R, SR>) {
-  let sortText = '';
-  if (sortDirection === 'ASC') {
-    sortText = '\u25B2';
-  } else if (sortDirection === 'DESC') {
-    sortText = '\u25BC';
-  }
-
   return (
     <span className={headerSortCellClassname} onClick={(e) => onSort(e.ctrlKey)}>
       <span className={headerSortNameClassname}>{children}</span>
       <span>
-        {sortText}
+        {sortDirection !== undefined && (
+          <svg viewBox="0 0 12 8" width="12" height="8" className={arrowClassname}>
```

Comment:
```suggestion
          <svg viewBox="0 0 12 8" width="12" height="8" className={arrowClassname} aria-hidden="true">
```

Not sure if we want to set `focusable` to false as we do not support IE 11
https://adrianroselli.com/2021/04/sortable-table-columns.html#SVG

Reply:
https://github.com/adazzle/react-data-grid/blob/main/src/formatters/ToggleGroupFormatter.tsx#L49
Should we do the same here?

Reply:
I think so but I will double check.

Reply:
I think we can make the same change

Reply:
Done